pkg/runtimeproxy/server/docker/server.go

```diff
@@ -145,7 +145,7 @@ func (d *RuntimeManagerDockerServer) failOver(dockerClient proxyDockerClient) er
 			ContainerResourceHookRequest: &v1alpha1.ContainerResourceHookRequest{
```

Comment:
Does `ContainerEnvs:        splitDockerEnv(ContainerConfig.Env)`  also need to be added when failover?

Reply:
done